### ISSUE TITLE
CldMediaLibrary component issues

### ISSUE THREAD

**Issue by R0N1n-dev** [nuxt-modules/cloudinary#164]
The CldMediaLibrary component has issues. It is not working 

### Version

"@nuxtjs/cloudinary": "^2.4.1",
"nuxt": "^3.8.0"

### Steps to reproduce
Install module as on docs page
Tried to use CldMediaLibrary component as in docs
### What is Expected?
It works and shows media library
### What is actually happening?
![Screenshot from 2023-11-20 12-02-12](https://github.com/nuxt-modules/cloudinary/assets/83344872/1045f424-a6f6-4bd8-be2b-bd9b497daace)
![Screenshot from 2023-11-20 12-22-12](https://github.com/nuxt-modules/cloudinary/assets/83344872/682384dd-c0c7-40e4-9b6d-d04b1b38fda6)

**Comment by Baroshem** [comment]
Hey, 

Thanks for reporting this issue.

I have some questions however because it does not occur on my local environment nor on the deployed documentation as you can see on the attached screenshot

![image](https://github.com/nuxt-modules/cloudinary/assets/37120330/c58de061-091a-4dff-9e46-60905bf83369)

Also, the error message suggests that there is no `PropType` export in `vue` plugin for Vite which is not fully related to the module. 

PropType is used in ProductGallery (which works on the documentation as well) and MediaLibrary.

**Comment by Baroshem** [comment]
I have also just tested it on a fresh nuxt 3.8.0 project with NuxtCloudinary 2.4.1 and the MediaLibrary works as expected without errors.

**Comment by R0N1n-dev** [comment]
New project
![Screenshot from 2023-11-20 12-51-28](https://github.com/nuxt-modules/cloudinary/assets/83344872/a3a7be7a-5f8e-44a5-8569-9ac02ebaec66)
 So can i borrow ur pc

**Comment by Baroshem** [comment]
I just checked this on stackblitz with 3.8.0 and NuxtCloudinary 2.4.1 and it is showing this bug. You are correct.

I will publish a fix in a sec

**Comment by Baroshem** [comment]
Can you check on 2.4.2 version?

**Comment by R0N1n-dev** [comment]
Is it supposed to look like this
![Screenshot from 2023-11-20 14-50-39](https://github.com/nuxt-modules/cloudinary/assets/83344872/cb400daa-ace6-4e17-81c0-5afb759fdcee)

**Comment by Baroshem** [comment]
I have just checked the version 2.4.3 (I needed to publish one more patch version due to the bug in import of types from Vue package for some reason) on stackblitz with nuxt 3.8.0 and it is working correctly.

**Comment by Baroshem** [comment]
> Is it supposed to look like this ![Screenshot from 2023-11-20 14-50-39](https://user-images.githubusercontent.com/83344872/284256639-cb400daa-ace6-4e17-81c0-5afb759fdcee.png)

Yes, sorry for the trouble! As I mentioned, when testing locally in several scenarios this bug was not appearing (even in a fresh nuxt app)

**Comment by R0N1n-dev** [comment]
Its rather short and cant adjust it, cant see any images???

**Comment by Baroshem** [comment]
You can style it with `#widget_container`.

We can also add a feature request to allow for passing the styles directly to component as class or style. Currently it is inside a div but we can modify it for the next version. If you think that it would be useful, please create an issue and we will plan it accordingly for new versions.

I am closing this issue as the bug was resolved.

**Comment by R0N1n-dev** [comment]
Nope, I cannot style it except in the console. Anything applied in style tags not working.
![Screenshot from 2023-11-20 15-35-10](https://github.com/nuxt-modules/cloudinary/assets/83344872/ea974dbf-f65a-4d32-9099-9f09ea06f886)

Had to add it in console
![Screenshot from 2023-11-20 15-37-37](https://github.com/nuxt-modules/cloudinary/assets/83344872/ff0215e6-9084-4483-8d4b-f674f098f830)

Styling with style tags is broken for some reason

**Comment by colbyfayock** [comment]
here's a codesandbox demo that you can add your cloud name and API key to see what it looks like outside of the context of Nuxt for help debugging: https://codesandbox.io/s/media-library-widget-b7hgbg?from-embed

one thing to note is it looks like this is currently embedding it on the page, instead of opening it in a modal, is that the desired experience?

if that is the expectation, the parent container would need to be tall enough to show the entirety of it or have a flexible height that would allow it to expand completely, which depending on your layout could interfere with that

if you want it to open in a modal, `inline_container` should not be applied. i would expect opening in a modal would be the "happy path" and default experience of this, much like the Upload Widget

**Comment by R0N1n-dev** [comment]
Dont know what to do,still cant style it except in console.

**Comment by Baroshem** [comment]
Hey @R0N1n-dev 

I am currently fixing that to allow you to pass some custom styling to the element directly. I was just busy with a release of another module.

New version (2.5.0) will be released this week :)

**Comment by R0N1n-dev** [comment]
> Hey @R0N1n-dev
> 
> I am currently fixing that to allow you to pass some custom styling to the element directly. I was just busy with a release of another module.
> 
> New version (2.5.0) will be released this week :)

Tried V2.5.1 and  still same result. Styling in console still only and useless option.
Thanxxxx alot again

**Comment by Baroshem** [comment]
Hey @R0N1n-dev 

You are correct. While developing other features for this release, I missed adding this fix. It should be now working with (2.5.2). 

With this change, you should be able to style the component like following:

```html
  <CldMediaLibrary
    api-key="12345"
    :button-id="buttonId"
    style="height: 600px"
  />
```

Please remember that Open Source projects are always open for contributions. If you see something that does not work correctly, you can create a PR with a fix for that and we will appreciate all the help.

Regarding the case of closing the issue. The issue you have created was about the fact that `CldMediaLibrary` was not working with the published npm package which I explained [here](https://github.com/nuxt-modules/cloudinary/issues/164#issuecomment-1818909320) and this was fixed with 2.4.3 version and that is why I closed the issue.

The next issue you are referring to, was related to styling the component to match your needs and for that kind of situations it is better to open a new issue or a question as the previous issue was resolved.
